oci/oci-how-to/getting-started.rst

```diff
@@ -0,0 +1,44 @@
+Getting Started
+***************
+
+This is where to find the Ubuntu container images:
+
+.. tabs::
+
+	.. tab:: Docker Hub
+
+         .. code-block:: bash
+
+            docker pull ubuntu:latest
+            
+
+	.. tab:: Amazon ECR
+
+         .. code-block:: bash
+            
+            docker pull public.ecr.aws/ubuntu/ubuntu:latest
+
+	.. tab:: Microsoft ACR
+
+         .. code-block:: bash
+            
+            docker pull ubuntu.azurecr.io/ubuntu:latest
+
+	.. tab:: IronBank
+
+         .. note::
+            Please note that these Ubuntu container images may have special terms of use and fall under Ubuntu Pro. Please reach out if you would like to use the Ubuntu container images from IronBank.
```

Comment:
Who/where should users reach out to in this case? Could we add a link?

Reply:
We already have some links in the footer and the Project into at the index page.

I'm adding an email link nonetheless